src/mudlet.h

```diff
@@ -233,7 +233,7 @@ public slots:
     void slot_multi_view();
     void slot_stopAllTriggers();
     void slot_userToolBar_hovered(QAction* pA);
-    void slot_connection_dlg_finnished(const QString& profile, int historyVersion);
+    void slot_connection_dlg_finished(const QString &profile, int historyVersion);
```

Comment:
Accidentally changed by IDEs own autoindenting - thanks for picking it up, I've re-ran clang-format.